src/main/java/org/broadinstitute/dsm/jobs/TestBostonUPSTrackingJob.java

```diff
@@ -436,15 +438,16 @@ private void updateTrackingInfo(PoolingDataSource<PoolableConnection> dataSource
                         if (earliestInTransitTime != null && !kit.getCE_order()) {
                             // if we have the first date of an inbound event, create an order in CE
                             // using the earliest date of inbound event
-                            Authentication careEvolveAuth = null;
-                            if (orderRegistrar == null) {
+
+                            if (orderRegistrar == null || careEvolveAuth == null) {
```

Comment:
@zyme here is the change

Reply:
You can see we added the `careEvolveAuth` to the condition and initialization was moved to L43